packages/app-btc/src/utils/operationHelper.ts

```diff
@@ -63,4 +66,57 @@ export class OperationHelper<Q extends QueryKey, R extends ResultKey> {
     parseCommonError((result[this.resultKey] as any).commonError);
     return resultData;
   }
+
+  private static splitIntoChunks(txn: Uint8Array): Uint8Array[] {
+    const chunks: Uint8Array[] = [];
+    const totalChunks = Math.ceil(txn.length / OperationHelper.CHUNK_SIZE);
+
+    for (let i = 0; i < totalChunks; i += 1) {
+      const chunk = txn.slice(
+        i * OperationHelper.CHUNK_SIZE,
+        i * OperationHelper.CHUNK_SIZE + OperationHelper.CHUNK_SIZE,
+      );
+      chunks.push(chunk);
+    }
+
+    return chunks;
+  }
+
+  public async sendInChunks<
+    RK extends keyof Exclude<Result[R], null | undefined>,
+    QK extends keyof Exclude<Query[Q], null | undefined>,
+  >(data: Uint8Array, queryKey: QK, resultKey: RK) {
+    const chunks = OperationHelper.splitIntoChunks(data);
+    let remainingSize = data.length;
+
+    for (let i = 0; i < chunks.length; i += 1) {
+      const chunk = chunks[i];
+      remainingSize -= chunk.length;
```

Comment:
I don't think we need to send in chunks in this PR

Reply:
that was part of latest proto, it's split between 2 commits, this commit `8980e19419f43e7cd64a44edac81cc949cbee032` only contains old proto changes

Reply:
we can open another pr from just that commit if we need to skip chink changes.